Added Myself (Yash Verma)

YashKumarVerma opened the pull request:
Google Codein Task

YashKumarVerma committed:
Added Myself (Yash Verma)

Google Codein Task

prwatch commented:
DEMO: http://YashKumarVerma.github.io/gci15.fossasia.org

YashKumarVerma committed:
Update README.md

YashKumarVerma commented:
Why is my pull-request nor merged ?

maitreyav commented:
Hi @YashKumarVerma, I also had the same issue. Refer [this.](https://github.com/fossasia/gci15.fossasia.org/blob/gh-pages/GIT.md)

roonyh commented on _data/students.yml:
```diff
@@ -341,3 +341,11 @@
+  github: YashKumarVerma
+  image: yash_kumar_verma.JPG
+  facebook: http://www.facebook.com/yashkumarverma
+  twitter: @yash_kr_verma
```
Don't add the `@`

roonyh commented:
@YashKumarVerma Its because the page has errors. Remove the `@` mark and it should be fine.

YashKumarVerma committed:
Removed @ sign from students.yml

YashKumarVerma committed:
Updated link

YashKumarVerma committed:
Added Thanks for Contributing

Yash Kumar Verma

YashKumarVerma commented:
- Added Myself
- Added Thanks for Contributing
- Made a table around Loklak peers
- Added My Project To Gallery
- _Fixed Improper (") in thanks for contributing script.

please merge as soon as possible.

Thanks in advance.

jig08 commented:
@YashKumarVerma Resolve merge conflicts.

YashKumarVerma commented:
um... may you please tell me. I am a beginner here

YashKumarVerma commented:
ok. I am closing this pull request. I will do the entire process again. Thanks for help guys.
